### ISSUE TITLE
Count SLOC for JSON files

### ISSUE THREAD

**Issue by aayushbhargava** [edvinh/node-sloc#14]
I guess, the library doesn't support `.json` file extension at the moment.
Can someone help with this issue, & if we support can some one help with that. 

Because when i run with the option config :
```
{
	path: 'src',
	extensions: ['json'],
	ignorePaths: ['node_modules'],
	ignoreDefault: false,
}
```

Running I'm getting this error :
**### Error when walking directory: TypeError: Reduce of empty array with no initial value**

**Comment by edvinh** [comment]
Hi! I managed to reproduce the issue. It only happens when the target path has no files, so a workaround for now would be to make sure the folder is not empty (`src` in this case). I have a fix for it already, a patch will be published as soon as possible.

**Comment by edvinh** [comment]
Fixed in 0.2.1. Closing this issue.
